charts/k8sgpt/values.yaml

```diff
@@ -0,0 +1,25 @@
+deployment:
+  image: ghcr.io/k8sgpt-ai/k8sgpt:v0.2.4
```

Comment:
Can we add an annotation for release-please here?

Reply:
@AlexsJones
```yaml
  labels:
    {{- include "k8sgpt.labels" . | nindent 4 }}
```
Will bring in `Chart.version` and `Chart.appVersion` automatically. I think that addresses this comment, though please correct me if I am misunderstanding!

I'll add user-specified annotations though.

Reply:
@AlexsJones ok now I understand you're talking about [release-please](https://github.com/googleapis/release-please), but still unsure how to structure the annotation.